easybuild/easyconfigs/q/QuantumESPRESSO/QuantumESPRESSO-6.3-intel-2018b.eb

```diff
@@ -0,0 +1,56 @@
+name = 'QuantumESPRESSO'
+version = '6.3'
+
+homepage = 'http://www.pwscf.org/'
```

Comment:
@akesandgren This domain is now dead, please fix to https://www.quantum-espresso.org